### PR TITLE
RHV Provider playbooks and role

### PR DESCRIPTION
**Pull request by cwilkers**
RHV Provider playbook directory and `openshift_rhv` role

* Provisions VMs on RHV
* Provides DNS update files
* Uninstall plays

CC @cooktheryan @sdodson PTAL

### CODE REVIEW

**Comment by cwilkers** [comment]
The travis issue stems from use of included roles, but there appears to be no mechanism to include those roles into CI as a requirement.

**Comment by cwilkers** [comment]
WIP: Changing generate_dns code into generate_nsupdate and ensuring it uses the Ansible module for nsupdate.

**Comment by sdodson** [comment]
Lets vendor the role dependencies into our repo.

**Comment by cwilkers** [comment]
bot, retest this please

File: playbooks/rhv/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### Let's Provision!
+#### High-level overview
+After populating inventory and variables files with the proper values,
+(see [The OpenShift Advanced Installation Documentation](https://docs.openshift.com/container-platform/latest/install_config/install/advanced_install.html)
+) a series of Ansible playbooks from this subdirectory will provision a set of
+nodes on the RHV (or oVirt) cluster, prepare them for OpenShift installation,
+and deploy an OpenShift cluster on them. If supplied with an *nsupdate* server
+and *rndc* key, the entire installation may be performed unattended.
+
+#### Step 1 Inventory
```
Comment (jirib) [inline playbooks/rhv/README.md]: This section is missing content.

**Comment by cwilkers** [comment]
WIP: I'm rebasing this work and will generate a new PR.

**Comment by cwilkers** [comment]
Closing due to change in how we're handling oVirt roles.
Will be rebased into https://github.com/openshift/openshift-ansible/pull/9570

**Pull request closed without merging**